tests/unit/App.spec.js

```diff
@@ -89,4 +94,74 @@ Given("an App", () => {
       expect(app.vm.url).to.equal(expectedUrl);
     });
   });
+
+  When("a successful response is returned", () => {
+    let codeProps;
+
+    beforeEach(async () => {
+      app.setData({
+        body: "some-body",
+        wasResponseAnError: false,
+        statusText: ""
+      });
+
+      app.setMethods({
+        request: app.vm.handleResponse
+      });
+
+      app.vm.request(successfulResponse);
+
+      codeProps = app.vm.$children[1]._props;
```

Comment:
what happens if you have more children? would this change?

Reply:
I've added in a `ref` for the Code component, which allows me to refer to that specific component.

Reply:
I'd advise against writing code just to make your tests work. Can you do this without adding the ref?

Reply:
I've added a helper function that goes through `$children` and creates a map of component names (`UrlBar`) to the properties of that component. This is better than using a `ref`, but I'm not sure if it is what you were thinking.